media-gfx/blender-bin/blender-bin-3.6.19.ebuild

```diff
@@ -7,67 +7,166 @@ inherit desktop xdg-utils
 
 DESCRIPTION="3D Creation/Animation/Publishing System"
 HOMEPAGE="https://www.blender.org"
+
 LICENSE="GPL-3+ Apache-2.0"
-SLOT="${PV%.*}"
-RESTRICT="strip"
+SLOT="$(ver_cut 1-2)"
 
-if [[ ${PV} = *9999* ]] ; then
-	BDEPEND="
-		app-misc/jq
-	"
-	PROPERTIES+=" live"
+if [[ ${PV} == *9999* ]] ; then
+	PROPERTIES="live"
 else
 	SRC_URI="
 		https://download.blender.org/release/Blender${SLOT}/blender-${PV}-linux-x64.tar.xz
 	"
 	KEYWORDS="~amd64"
 fi
 
+IUSE="oneapi"
+RESTRICT="strip test"
+
+QA_PREBUILT="opt/${P}/*"
+
+if [[ ${PV} == *9999* ]] ; then
+	BDEPEND="
+		app-misc/jq
+	"
+fi
+
+# no := here, this is prebuilt
 RDEPEND="
-media-libs/libglvnd
-sys-libs/glibc
-virtual/libcrypt
-x11-base/xorg-server
+	app-arch/zstd
+	media-libs/libglvnd[X]
+	sys-libs/glibc
+	sys-libs/ncurses
+	sys-libs/zlib
+	virtual/libcrypt
+	x11-base/xorg-server
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXi
+	x11-libs/libXrender
+	x11-libs/libXt
+	x11-libs/libXxf86vm
+	x11-libs/libxkbcommon
+	oneapi? (
+		dev-libs/level-zero
+	)
 "
 
 src_unpack() {
-	mkdir -p "${S}"
-	if [[ ${PV} = *9999* ]] ; then
-		if [[ -z "${BLENDER_BIN_URL}" ]]; then
-			latest=$(wget -O - "https://builder.blender.org/download/daily/?format=json&v=1" | \
-				jq -r 'map(select(.platform == "linux" and .branch == "main" and .file_extension == "xz")) | .[0].url' )
+	local my_A
+	if [[ ${PV} == *9999* ]] ; then
+		# TODO what does BLENDER_BIN_URL mean?
```

Comment:
Do you want me clarify the variable name?

It meant to be used to check out different PR branches or forks of Blender that provides binary packages.

Reply:
Where should we put this explaination. Perhaps just a longer comment in the ebuild?